online-shop/src/app/components/presentational/product-details-view/product-details-view.component.html

```diff
@@ -0,0 +1,29 @@
+<body class="body">
```

Comment:
Use only divs, tag <body> shouldn't be in the html file of a component

Reply:
Made it a div